bucket4j/src/main/java/com/linecorp/armeria/common/throttling/ThrottlingHeadersImpl.java

```diff
@@ -37,32 +37,19 @@
     }
 
     private static AsciiString createHeaderName(String name) {
-        return AsciiString.cached(Ascii.toLowerCase(name));
+        return HttpHeaderNames.of(name);
     }
```

Comment:
I guess you can just inline `createHeaderName` now that it's just a single call.

Reply:
Fixed